drupal/content.md

```diff
@@ -0,0 +1,50 @@
+
+> [wikipedia.org/wiki/Drupal](https://en.wikipedia.org/wiki/Drupal)
+
+%%LOGO%%
+
+# How to use this image
+
+The basic pattern for starting a `%%REPO%%` instance is:
+
+	docker run --name some-%%REPO%% -d %%REPO%%
+
+If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
+
+	docker run --name some-%%REPO%% -p 8080:80 -d %%REPO%%
+
+Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+
+There are multiple database types supported by this image, most easily used via standard container linking. In the default configuration, SQLite can be used to avoid a second container and write to flat-files. More detailed instructions for different (more production-ready) database types follow.
+
+When first accessing the webserver provided by this image, it will go through a brief setup process. The details provided below are specifically for the "Set up database" step of that configuration process.
+
+## MySQL
+
+	docker run --name some-%%REPO%% --link some-mysql:mysql -d %%REPO%%
+
+-	Database type: `MySQL, MariaDB, or equivalent`
+-	Database name/username/password: `<details for accessing your MySQL instance>` (`MYSQL_USER`, `MYSQL_PASSWORD`, `MYSQL_DATABASE`; see environment variables in the description for [`mysql`](https://registry.hub.docker.com/_/mysql/))
+-	ADVANCED OPTIONS; Database host: `mysql` (for using the `/etc/hosts` entry added by `--link` to access the linked container's MySQL instance)
+
+## PostgreSQL
+
+	docker run --name some-%%REPO%% --link some-postgres:postgres -d %%REPO%%
+
+-	Database type: `PostgreSQL`
+-	Database name/username/password: `<details for accessing your PostgreSQL instance>` (`POSTGRES_USER`, `POSTGRES_PASSWORD`; see environment variables in the description for [`postgres`](https://registry.hub.docker.com/_/postgres/))
+-	ADVANCED OPTIONS; Database host: `postgres` (for using the `/etc/hosts` entry added by `--link` to access the linked container's PostgreSQL instance)
+
+## Adding additional libraries / extensions
+
+This image does not provide any additional PHP extensions or other libraries, even if they are required by popular plugins. There are an infinite number of possible plugins, and they potentially require any extension PHP supports. Including every PHP extension that exists would dramatically increase the image size.
+
+If you need additional PHP extensions, you'll need to create your own image `FROM` this one. The [documentation of the `php` image](https://github.com/docker-library/docs/blob/master/php/README.md#how-to-install-more-php-extensions) explains how to compile additional extensions.
```

Comment:
Also link to it's own Dockerfile as an example? https://github.com/docker-library/drupal/blob/master/7/Dockerfile#L6-9